main.tf

```diff
@@ -68,6 +68,8 @@ resource "aws_security_group_rule" "docker_machine_docker_self" {
 }
 
 resource "aws_security_group_rule" "docker_machine_ssh" {
+  count = var.docker_machine_ssh_restrict_ingress_to_runner ? 0 : 1
```

Comment:
@bishtawi I like the work you did to get it backwards compatible. But your change is not breaking anything at all. I would suggest the we remove the variable `docker_machine_ssh_restrict_ingress_to_runner` and remove the cidr rule as well.